packages/redux/src/themes/__tests__/selectors.test.ts

```diff
@@ -0,0 +1,92 @@
+import * as fromReducer from '../reducer.js';
+import * as selectors from '../selectors.js';
+import {
+  mockState,
+  mockTheme,
+  mockThemeCode,
+} from 'tests/__fixtures__/themes/index.mjs';
+import { toBlackoutError } from '@farfetch/blackout-client';
+
+describe('Themes selectors', () => {
+  beforeEach(jest.clearAllMocks);
+
+  describe('isThemesLoading()', () => {
+    it('should get the loading status', () => {
+      const spy = jest.spyOn(fromReducer, 'getIsLoading');
+
+      expect(selectors.isThemeLoading(mockState, mockThemeCode)).toBe(false);
+      expect(spy).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe('getThemesError()', () => {
```

Comment:
```suggestion
  describe('getThemeError()', () => {
```

Review other descriptions in this file as well as they are mismatched with the selector to be tested.

Reply:
Done